improver/cli/wxcode.py

```diff
@@ -63,6 +63,9 @@ def process(*cubes: cli.inputcube, wxtree="high_resolution"):
             weather symbols decision tree, these at co-incident times.
         wxtree (str):
             Weather Code tree: high_resolution or global.
+        model_id_attr (str):
+            Name of attribute recording source models. The source models are
```

Comment:
I think we need to describe the behaviour this argument will invoke. How about
```suggestion
            Name of attribute recording source models that should be inherited by the output cube. The source models are
```

Reply:
Amended as suggested.